rigex-front-end/src/constants/typeDefs.js

```diff
@@ -46,6 +47,18 @@ export const typeDefs = gql`
     text: String!
   }
 
+  type Tests {
```

Comment:
why does it have a plural name? Should it be singular? Or is the array of tests itself the item being referred to?